packages/cli-kit/src/public/node/system.ts

```diff
@@ -16,6 +16,7 @@ export interface ExecOptions {
   stdio?: 'inherit'
   input?: string
   signal?: AbortSignal
+  externalErrorHandler?: (error: unknown) => Promise<void>
```

Comment:
we should document what this is for (vs. wrapping the call to `exec` in a `try/catch`